lib/slim_fast/parser.ex

```diff
@@ -95,6 +71,41 @@ defmodule SlimFast.Parser do
     end
   end
 
+  @attr_regex ~r/([\w-]+)\s*=\s*(".+"|[\w-]+)/
+  defp parse_attributes(line, acc \\ [])
+  defp parse_attributes("", acc), do: {"", acc}
+  defp parse_attributes(nil, acc), do: {"", acc}
+  defp parse_attributes("(" <> line, acc), do: unwrap_attributes(line)
```

Comment:
I would suggest to split this function into 2: `parse_attributes/1`, which handles nil and deals with attributes wrapping and after that calls `parse_attributes/2` which parses attributes and accumulate the result.
Now it is unclear why `acc` is not passed in nested call.